config/DELFI.BB-flex/location_groups.txt

```diff
@@ -0,0 +1,126 @@
+location_group_id,location_id,location_group_name
+flex-476-stops,de:12073:900340004,durch den RufBus 476 bedientes Gebiet im Raum Angermünde
```

Comment:
OTP requires, that locations of a location_group can't be stations. Could you remove this, please. 
Note: The spec itself does not explicity name the allowed location_type, only speaks of "stops"

Reply:
🤦‍♂️ – I had already fixed that, but I didn't commit it.